wandb/sdk/launch/agent/agent.py

```diff
@@ -228,6 +211,49 @@ def __init__(self, api: Api, config: Dict[str, Any]):
         self._name = agent_response["name"]
         self._init_agent_run()
 
+    def _log(self, level: str, message: str):
```

Comment:
is the `level` actually a str or it is a literal that can be `ERROR`, `WARN`, `INFO`, `DEBUG`?

Reply:
also why not just define a new logger class that has a few methods and just call these methods seems to be more readable

Reply:
l = newLoger(verbosity)
l.debug(...) etc

Reply:
the other option requires more build up tho